frontend/src/routes/manage/Realm/Content/Edit/EditMode/Video.tsx

```diff
@@ -96,19 +103,34 @@ export const EditVideoBlock: React.FC<EditVideoBlockProps> = ({ block: blockRef
             )}
         />
 
-        <Heading>{t("manage.realm.content.titled.title")}</Heading>
-        <label>
-            <input
-                type="checkbox"
-                defaultChecked={showTitle}
-                {...form.register("showTitle")}
-                css={{ marginRight: 6 }}
-            />
-            {t("manage.realm.content.titled.show-title")}
-        </label>
+        <div css={{ display: "flex", flexDirection: "row", gap: 16, marginTop: 8 }}>
+            {optionProps.map(([option, checked, title]) =>
+                // eslint-disable-next-line react/jsx-key
+                <DisplayOption form={form} option={option} checked={checked} title={title} />)
```

Comment:
```suggestion
                <DisplayOption {...{ form, option, checked, title }} />)
```

Maybe slightly nicer?